app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <resources>
-    <string name="app_name">SampleApp</string>
-    <string name="title_activity_main">MainActivity</string>
+    <string name="app_name">PocketTutor</string>
+    <string name="title_activity_main">PocketTutor</string>
```

Comment:
Good catch! This could be misleading if the name of the app is not clear but now it is.